All_Relationships/Experiment 1/LSTM.py

```diff
@@ -0,0 +1,55 @@
+import csv
+import os
+
+import numpy as np
+import tqdm
+from snorkel.learning.disc_models.rnn import reRNN
+from snorkel.learning.disc_models.rnn.utils import SymbolTable
+
+
+def read_csv_file(filename, get_max_val=False):
```

Comment:
This needs a docstring for what kind of CSV it reads and what it is doing with the max int operations